### PR TITLE
feat(playground): visualize scope

### PR DESCRIPTION
**Pull request by Dunqing**
Partial support https://github.com/oxc-project/oxc/issues/1048

image

### CODE REVIEW

**Comment by Dunqing** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1882** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1882?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1882?utm_source=stack-comment).

File: website/playground/index.js [hunk 1]
```diff
@@ -575,6 +581,7 @@ query {
         break;
     }
 
+    console.log(text)
```
Comment (Boshen) [inline website/playground/index.js:584]:
```suggestion
```

File: website/playground/index.js [hunk 2]
```diff
@@ -539,11 +545,11 @@ class Playground {
...
-      case "format":
-        this.runOptions.format = true;
-        this.run();
-        text = this.oxc.formattedText;
-        break;
+        case "format":
+          this.runOptions.format = true;
+          this.run();
+          text = this.oxc.formattedText;
+          break;
```
Comment (Boshen) [inline website/playground/index.js:552]: broken indentation?

**Review by Boshen — CHANGES_REQUESTED** [review]

File: website/playground/index.js (hunk 2 above)
Comment (Dunqing) [reply website/playground/index.js]: I accidentally added indentation

**Review by Boshen — APPROVED** [review]

**Pull request merged**